app/components/primer/alpha/tool-tip-element.ts

```diff
@@ -175,14 +175,19 @@ class ToolTipElement extends HTMLElement {
     return this.ownerDocument.getElementById(this.htmlFor)
   }
 
-  connectedCallback() {
+  // eslint-disable-next-line custom-elements/no-constructor
```

Comment:
I've filed an issue to change this rule -- https://github.com/github/eslint-plugin-custom-elements/issues/48